### ISSUE TITLE
Back-end Changes

### ISSUE THREAD

**Issue by DustinMartin95** [revaturelabs/cms-meta#19]
@Quinn-Donnelly 
You mentioned on Monday about having to make some changes for the back-end as it is currently not working in the environment. Do you have more info on exactly what changes are needed?

**Comment by Quinn-Donnelly** [comment]
@DustinMartin95 Yes I will write out the details to that in a moment. Should have time to do so over lunch.

**Comment by DustinMartin95** [comment]
@Quinn-Donnelly 
Is there an update on these details?

**Comment by Quinn-Donnelly** [comment]
#21 is the issue in questions it was in the wrong repo.
